README.md

```diff
@@ -689,6 +689,29 @@ See [full report](Code4rena%20CSM%20V2%20Audit%20Report%2008-2025.pdf) for more
 
 See [full report](MixBytes%20WstETH%20Staker%20Security%20Audit%20Report%2009-2025.pdf) for more details.
 
+### 09-2025 Ackee Blockchain Lido Triggerable Withdrawals Security Audit
+
+- Total Issues: 11 (9 Fixed, 1 Partially fixed, 1 Acknowledged)
+- Low Issues: 2 (2 Fixed)
+- Warning Issues: 4 (3 Fixed, 1 Acknowledged)
+- Info Issues: 5 (4 Fixed, 1 Partially fixed)
+
+See [full report](Ackee%20Blockchain%20Lido%20Triggerable%20Withdrawals%20Audit%20Report%2009-25.pdf) for more details.
+
+### 09-2025 Composable Security Lido Oracle v6 Security Audit
```

Comment:
<img width="1096" height="680" alt="image" src="https://github.com/user-attachments/assets/a6b277ec-5c46-41da-b32f-d0c3037ec4cc" />

would suggest to polish the report's content alignment (not a blocker now, can be a separate PR when this is merged)

Reply:
Replaced with a better formatted version